examples/jsm/exporters/GLTFExporter.js

```diff
@@ -1153,10 +1153,10 @@ GLTFExporter.prototype = {
 		 */
```

Comment:
Most files in examples/**jsm** are generated automatically from a source version in examples/**js**, including GLTFExporter. Could you apply this change to the other copy? You can then either omit the `examples/jsm` version from your PR, or generate it yourself with `node utils/modularize.js`.

Reply:
Hello @donmccurdy, thank you for commets, I will update PR in the couple of days